src/views/Give/RecipientModal.jsx

```diff
@@ -49,7 +50,7 @@ export function RecipientModal({ isModalOpen, callbackFunc, cancelFunc, currentW
    * TODO consider extracting this into a helper file
    */
   const sohmBalance = useSelector(state => {
-    return state.account.balances && state.account.balances.sohm;
+    return state.account.balances && state.account.balances.mockSohm;
```

Comment:
This is the kind of change that will be forgotten about and will make it into production. 😰